tests/kafkatest/tests/tools/replica_verification_test.py

```diff
@@ -20,7 +20,6 @@
 from ducktape.tests.test import Test
 
 from kafkatest.services.verifiable_producer import VerifiableProducer
-from kafkatest.services.zookeeper import ZookeeperService
 from kafkatest.services.kafka import KafkaService, quorum
 from kafkatest.services.replica_verification_tool import ReplicaVerificationTool
 
```

Comment:
Could you also rename `num_zk` this field?

Reply:
https://issues.apache.org/jira/browse/KAFKA-18582
This is tracking by this issue.